src/sim65/6502.c

```diff
@@ -362,6 +389,21 @@ static void OPC_6502_01 (void)
 
 
 
+static void OPC_65SC02_04 (void)
+/* Opcode $04: TSB zp */
+{
+    unsigned char ZPAddr;
+    unsigned char Val;
+    Cycles = 5;
+    ZPAddr = MemReadByte (Regs.PC+1);
+    Val    = MemReadByte (ZPAddr);
```

Comment:
The style of this file is to not have alignments of consecutive assignments (and you don't have them at other newly added code).